odl/space/space_utils.py

```diff
@@ -203,6 +206,233 @@ def rn(size, dtype=None, impl='numpy', **kwargs):
     return rn
 
 
+class auto_weighting(OptionalArgDecorator):
+
+    """Make an unweighted adjoint automatically account for weightings.
```

Comment:
We need to clearly define what "unweighted" means here. For discrete lp spaces, is it the "unit weighting", or is it the "weight 1 for each pixel" weighting so to say?

Reply:
Done that, please check the updated docs.